indico/modules/events/clone.py

```diff
@@ -213,3 +214,25 @@ def run(self, new_event, cloners, shared_data, event_exists=False):
             # placeholder-like stuff in the pattern
             new_event.title = series.event_title_pattern.replace('{n}', str(n))
         db.session.flush()
+
+
+class EventPrivacyCloner(EventCloner):
+    name = 'event_privacy'
+    friendly_name = _('Privacy settings')
+    is_default = True
+
+    @property
+    def is_available(self):
+        return self._has_content(self.old_event)
+
+    def _has_content(self, event):
+        privacy_settings_data = privacy_settings.get_all(event, no_defaults=True)
+        return bool(next((v for v in privacy_settings_data.values() if v), False))
```

Comment:
Would this also work?

```suggestion
        return any(privacy_settings_data.values())
```

Reply:
Yes, changed.